crates/sound/src/audio_backends/alsa.rs

```diff
@@ -490,9 +497,9 @@ impl AlsaBackend {
                             st.params.format = request.format;
                             st.params.rate = request.rate;
                         }
+                        // Manually drop msg for faster response: the kernel has a timeout.
+                        drop(msg);
```

Comment:
Is this change intended?
Maybe better to describe it in the commit or split it into another commit (we would have to rework the story, so not splitting it is fine with me).

Reply:
Yes, the commit's title is `sound: clean up and improve comments, definitions` (or did it end up somewhere else? I'll check)

Reply:
To specify, the clean up here is to improve readability, the block was for early dropping the streams mutex and it could need a double take if someone sees them separated while they are meant to be performed together, logically